cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1883,6 +1883,47 @@
           {
             "Key": "Name",
             "Value": "Master"
+          },
+          {
+            "Key": "aws-parallelcluster-attributes",
+            "Value": {
+              "Fn::Sub": [
+                "${BaseOS}, ${Scheduler}, ${version}",
+                {
+                  "version": {
+                    "Fn::FindInMap": [
+                      "PackagesVersions",
+                      "default",
+                      "parallelcluster"
+                    ]
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "Key": "aws-parallelcluster-filesystem",
+            "Value": {
+              "Fn::Sub": [
+                "efs=${efs}, multiebs=${NumberOfEBSVol}, raid=${raid}",
+                {
+                  "efs": {
+                    "Fn::If": [
+                      "CreateEFSSubstack",
+                      "true",
```

Comment:
What if we use 0 and 1 instead of true and false? It might be easier to query, aggregate and plot the values, isn't it?

Reply:
changed